src/_pytest/pythonpath.py

```diff
@@ -0,0 +1,21 @@
+import sys
+from typing import List
+
+import pytest
+from _pytest.config import Config
+from _pytest.config.argparsing import Parser
+
+
+def pytest_addoption(parser: Parser):
+    parser.addini("pythonpath", type="pathlist", help="Add paths to sys.path")
```

Comment:
Let's use `type="paths"`; `pathlist` uses the legacy `py.path` which is going to be deprecated at some point.